package.json

```diff
@@ -54,6 +54,7 @@
     "chalk": "^1.0.0",
     "dot-prop": "^2.2.0",
     "get-stdin": "^5.0.1",
+    "lodash": "^4.11.1",
```

Comment:
Size is a concern, and if another package is using a different version of lodash then they'll be fragmented anyway. Can't always count on a dependency upstream using the exact same lodash.